Counterfeit Monkey Materials/Extensions/Dannii Willis/Ultra Undo.i7x

```diff
@@ -18,6 +18,8 @@ Global ultra_undo_needed = 0;
 	if ( res == 1 ) ! Failure
 	{
 		ultra_undo_needed = 1;
+		! Delete any leftover undo files from earlier sessions.
```

Comment:
What if this was just run at the beginning of the function? I can't think of any reason why it shouldn't be run in the successful code path.